src/wrapper-end.js

```diff
@@ -3,8 +3,11 @@
     
   var Reflect;
 
-  System = new SystemLoader();
-  System.constructor = SystemLoader;
+  System = System || {};
+  if (!System.loader) {
+    System.loader = new SystemLoader();
+    System.loader.constructor = SystemLoader;
+  }
```

Comment:
@guybedford I don't think this is correct. Tests are failing, and I think this is because cjsMode needs System not SystemLoader